tests/test_upload_manager.py

```diff
@@ -32,5 +37,56 @@ def test_upload_manager_switches_drivers():
     container.bind('StorageConfig', storage)
     container.bind('UploadDiskDriver', object)
     container.bind('UploadTestDriver', object)
-
     container.bind('UploadManager', UploadManager(container).driver('test'))
+
+
+def test_upload_manage_accept_files():
+    container = App()
+
+    container.bind('Application', application)
+    container.bind('StorageConfig', storage)
+    container.bind('UploadDiskDriver', UploadDiskDriver)
+
+    class ImageTest():
+        """
+        Image test for emulate upload file
+        """
+
+        @property
+        def filename(self):
+            return os.getcwd() + "/tests/static/test.jpg"
+
+        @property
+        def file(self):
+            return open(self.filename, 'rb')
+
+    img = ImageTest()
+    UploadManager(container).driver('disk').accept('jpg', 'png').store(img)
```

Comment:
Also create a test without the `accept()` method and you'll find a bug that says `AttributeError: 'UploadDiskDriver' object has no attribute 'accept_file_types'`

Reply:
With this test working:

def test_upload_file():
    """
    This test is responsible for checking if you upload a file correctly.
    """

    container = App()

    container.bind('Application', application)
    container.bind('StorageConfig', storage)
    container.bind('UploadDiskDriver', UploadDiskDriver)

    img = ImageTest()
    UploadManager(container).driver('disk').store(img)